src/main/java/net/mcreator/element/types/Block.java

```diff
@@ -243,4 +248,11 @@ else if (renderType == 3)
 		return creativeTab;
 	}
 
+	@Override public @NotNull List<BoxEntry> getValidBoundingBoxes() {
+		return boundingBoxes.stream().filter(BoxEntry::isNotEmpty).collect(Collectors.toList());
+	}
+
+	public boolean allowsCustomBoundingBox() {
+		return blockBase == null || blockBase.equals("Leaves") || blockBase.equals("Pane");

```

Comment:
Is there any particular reason for leaves and pane here? Leaves can only be cube, and changing box of pane from its default system is not really something someone would want to do, I think.

Reply:
It's to keep the current behavior. Not sure why, but leaves and pane bases don't lock the bounding box options at the moment.

Reply:
Yep, noticed this was so before, I am not really sure why I made this so :D Well, it does not make much sense actually so you can remove these, but also make sure to disable the box editor in such cases.